src/main/java/Saved.java

```diff
@@ -0,0 +1,74 @@
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.FileWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class Saved {
+    private String filePath;
+
+    public Saved(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> loadData() throws FileNotFoundException, ParseException {
```

Comment:
It is normal practice to use the interface rather than the concrete class. For example, you could use List instead of ArrayList.